File: Difficulty: Medium/Missing Element in Range/missing-element-in-range.cpp
class Solution {
  public:
    vector<int> missinRange(vector<int>& arr, int low, int high) {
        // code here
        vector<int>ans;
        sort(arr.begin(),arr.end());
        for(int i=low;i<=high;i++)
        {
            int idx=lower_bound(arr.begin(),arr.end(),i)-arr.begin();
            if(idx==arr.size())
            {
                ans.push_back(i);continue;
            }
            if(arr[idx]!=i)
            {
                ans.push_back(i);
            }
        }
        return ans;
    }
};

File: Difficulty: Medium/Missing Element in Range/missing-element-in-range.java
class Solution {
    public ArrayList<Integer> missingRange(int[] arr, int low, int high) {
        // code here
        ArrayList<Integer>ls=new ArrayList<>();
        int[] dp = new int[100001];
        Integer n=arr.length;
        Arrays.sort(arr);
        for(int i=0;i<n;i++)
        {
          dp[arr[i]]++;  
        }
        for(int i=low;i<=high;i++)
        {
            if(dp[i]==0)
            {
                ls.add(i);
            }
        }
        return ls;
       
    }
}